### PR TITLE
Disable property wrapper composition.

### PR DESCRIPTION
**Pull request by jckarter**
There are ABI-affecting bugs with composition like SR-11138, so disable until those bugs can be
fixed. rdar://problem/53428736

### CODE REVIEW

**Comment by jckarter** [comment]
@swift-ci Please test

**Comment by jckarter** [comment]
@swift-ci Please test source compatibility

File: lib/Sema/TypeCheckPropertyWrapper.cpp [hunk 1]
```diff
@@ -462,10 +462,17 @@ AttachedPropertyWrappersRequest::evaluate(Evaluator &evaluator,
...
+    
     result.push_back(mutableAttr);
   }
 
+  // TODO: Property wrapper compositions do not yet compose correctly
+  // in all situtaions.
+  if (result.size() > 1) {
+    ctx.Diags.diagnose(result[result.size() - 2]->getLocation(),
+                       diag::property_wrapper_composition_not_implemented);
+  }
```
Comment (jrose-apple) [inline lib/Sema/TypeCheckPropertyWrapper.cpp]: Nitpick: If you do this after the `std::reverse` below, you can avoid the weird "second-to-last" computation above, which confused me until I read the comment below.

**Review by jrose-apple — APPROVED** [review]

**Comment by jckarter** [comment]
@swift-ci Please test

**Pull request merged**